satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -426,60 +426,61 @@ delete serial_number (
 // for preventing duplicate serial numbers
 create used_serial ()
 
-// --- bucket roll up tables --- //
+// --- bucket tally tables --- //
 
-model bucket_bandwidth_rollup (
+model bucket_bandwidth_tally (
 	key    bucket_id interval_start action
 	index (
-	    name bucket_id_interval_start_interval_seconds
-	    fields bucket_id interval_start interval_seconds
+	    name bucket_id_interval_start
+	    fields bucket_id interval_start
 	)
 
 	field bucket_id        blob
 	field interval_start   utimestamp
-	field interval_seconds uint
 	field action           uint
 
 	field inline    uint64
 	field allocated uint64
 	field settled   uint64
 )
 
-model bucket_storage_rollup (
+model bucket_storage_tally (
 	key    bucket_id interval_start
 
 	field bucket_id        blob
 	field interval_start   utimestamp
-	field interval_seconds uint
 
 	field inline uint64
 	field remote uint64
+	
+  field  remote_segments    uint
+  field  inline_segments    uint
+  field  objects            uint
+  field  metadata_size      uint64
 )
 
-// --- storage node roll up tables --- //
+// --- storage node tally tables --- //
 
-model storagenode_bandwidth_rollup (
+model storagenode_bandwidth_tally (
 	key    storagenode_id interval_start action
 	index (
-	    name storagenode_id_interval_start_interval_seconds
-	    fields storagenode_id interval_start interval_seconds
+	    name storagenode_id_interval_start
+	    fields storagenode_id interval_start
 	)
 
 	field storagenode_id   blob
 	field interval_start   utimestamp
-	field interval_seconds uint
```

Comment:
This is not redundant, I wasn't able to find to avoid this.

Reply:
Could you please explain why you weren't able to avoid this?

It seems like you could just calculate the duration of the interval from the `interval_start` of the previous record.